Sources/PathKit.swift

```diff
@@ -152,8 +152,15 @@ extension Path {
   ///
   public func abbreviate() -> Path {
 #if os(Linux)
-    // TODO: actually de-normalize the path
-    return self
+    let env = ProcessInfo.processInfo.environment
+    guard let home = env["HOME"], self.path.contains(home) else { return self }
+    let withoutHome = Path(self.path.replacingOccurrences(of: home, with: ""))
```

Comment:
What about if the path was the following: `/home/kyle/backups/home/kyle/some/file` and `HOME` was set to `/home/kyle`? This would incorrectly result in `~/backupssome/file`